src/components/SearchFilters.vue

```diff
@@ -340,10 +405,11 @@ export default {
     addFilter: function (filter) {
       //Do not set the value unless it is ready
       if (this.cascaderIsReady && filter) {
-        if (this.validateFilter(filter)) {
+        filter = this.validateAndConvertFilterToHierarchical(filter)
```

Comment:
The variable filter here should be renamed and defined here as it is named the same as the argument.

Reply:
fixed now :)